inginious/frontend/static/css/themes/Makefile

```diff
@@ -0,0 +1,16 @@
+BOOTSWATCH_URL	= https://github.com/thomaspark/bootswatch.git
+BASE_THEME	= yeti
+
+compile:
+	if [ ! -d "bootswatch" ]; then make install; fi
+	cd bootswatch; ./node_modules/grunt/bin/grunt swatch:$(theme)
+	mv bootswatch/dist/$(theme)/bootstrap.css $(theme).css
+
+install:
+	git clone $BOOTSWATCH_URL
```

Comment:
In this case we do not use a fixed version as discussed before. @nrybowski suggested to use a submodule to include the source at a given commit stage. We can probably add a `third-party` folder at the source root to store the submodule.

Besides, as discussed previously, it would probably be better to store the source files for yeti and the modified yeti dark mode into our repository (keeping their copyright of course) and only keep a submodule to bootstrap.

In all case we need to be able to rebuild the `dark_mode.css` file from the sources.